__tests__/app.test.js

```diff
@@ -311,5 +311,120 @@ describe("app", () => {
     });
   });
 
+  describe("PATCH /api/articles/:article_id", () => {
+  
+    test("status code: 200 and returns article with the updated votes when the vote input is positive", () => {
+      return request(app)
+      .patch('/api/articles/1')
+      .send({ inc_votes: 300 })
+      .expect(200)
+        .then((response) => {
+
+          expect(Object.keys(response.body).length).toBe(1);
+
+          const {updatedArticle} = response.body;
+
+          // to check the number of the properties
+
+          expect(Object.keys(updatedArticle).length).toBe(8);
+
+          // to check the accuracy of the updatedArticle data
+
+          expect(updatedArticle.votes).toBe(400);
+          expect(updatedArticle.article_id).toBe(1);
+          expect(updatedArticle.title).toBe('Living in the shadow of a great man');
+          expect(updatedArticle.topic).toBe('mitch');
+          expect(updatedArticle.author).toBe('butter_bridge');
+          expect(updatedArticle.body).toBe('I find this existence challenging');
+          expect(updatedArticle.created_at).toBe("2020-07-09T20:11:00.000Z");
+          expect(updatedArticle.article_img_url).toBe('https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700');
+        });
+    });
+
+    test("status code: 200 and returns article with the updated votes when the vote input is negative", () => {
+      return request(app).patch('/api/articles/1')
+      .send({ inc_votes: -50 })
+      .expect(200)
+        .then((response) => {
+          expect(response.body.updatedArticle.votes).toBe(50);
+        });
+    });
+
+    test("to get error 404 if the path is wrong", () => {
+      return request(app)
+        .patch('/api/wrongpath/1')
+        .expect(404)
+    });
```

Comment:
If you have the above test already for other endpoints you don't need it, it is not specific to the articles endpoint it is catching anything that does not match. You can have this test once and in it's own describe block to everything else 😄

Reply:
:D sure